src/core/utils/wrap_stdbool.h

```diff
@@ -44,7 +44,7 @@
 /* c++ has a built in bool */
 #else
 
-#if defined(_MSC_VER)
+#if HAVE__BOOL
```

Comment:
If you make this change then two things:

1. Remove the comment about VS at the `#endif` of this block.
2. I believe you need to update every VS project file to define this.